livekit-plugins/livekit-plugins-cartesia/livekit/plugins/cartesia/tts.py

```diff
@@ -53,15 +54,25 @@ def __init__(
         language: str = "en",
         encoding: TTSEncoding = "pcm_s16le",
         voice: str | list[float] = TTSDefaultVoiceId,
+        voice_controls: dict[str, any] | None = None,
```

Comment:
Is it possible to be more specific about these types?

Reply:
yes - with a new type definition. it looks like `"speed"` can be a signed number or a string, and `"emotion"` is an array of signed numbers or strings. No other keys currently supported. so we could add `speed: str | float | None` and `emotion: list[str | float] | None` instead of `voice_controls`, or we could define a new container type for them.

Reply:
I vote for explicit options with types. While it does require us to keep it up to date, it would help to avoid confusion when users put in incorrect values/types

Reply:
Yep makes sense. Updated to break them up. Speed got a proper type since only five strings are valid (or any number -1 to 1) but emotion didn't because each string is both an emotions and a degrees so there are 25 valid strings and I didn't think it made sense to enumerate.

I also added this to the test cases!